tests/base_conftest.py

```diff
@@ -55,10 +56,16 @@ def __init__(self, classic_contract, method_class=VyperMethod):
         self._classic_contract = classic_contract
         self.address = self._classic_contract.address
         protected_fn_names = [fn for fn in dir(self) if not fn.endswith("__")]
-        for fn_name in self._classic_contract.functions:
+
+        try:
+            fn_names = [fn["name"] for fn in self._classic_contract.functions._functions]
+        except web3.exceptions.NoABIFunctionsFound:
+            return
```

Comment:
I don't think we want to return nothing here.  Probably setting fn_names to an empty list makes most sense.